crates/rust-analyzer/src/config.rs

```diff
@@ -714,6 +716,19 @@ impl Config {
         }
     }
 
+    pub fn client_specific_adjustments(&mut self, client_info: &Option<ClientInfo>) {
+        // FIXME: remove this when we drop support for vscode 1.65 and below
+        if let Some(client) = client_info {
+            if client.name.contains("Code") || client.name.contains("Codium") {
+                if let Some(version) = &client.version {
+                    if version.as_str() < "1.76" {
```

Comment:
Shouldn't this be 1.75? 1.74 is the November release, they skip December, and 1.75 comes out in January.

Reply:
You are right. I wasn't aware that they will skip december release entirely.